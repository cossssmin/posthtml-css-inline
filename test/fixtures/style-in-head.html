<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }

    div {
      display: flex;
    }

    div {
      font-size: 14px !important;
    }

    @media screen {
      .sm\:w-1\/2 {
        width: 50%;
      }
    }

    .\@container{
      container-type: inline-size
    }

    @container (min-width: 17.5rem){
      .\@\[17\.5rem\]\:underline{
        text-decoration-line: underline
      }
    }

  </style>
</head>
<body>
  <section class="@container">
    <div class="text-blue text-sm @[17.5rem]:underline sm:w-1/2"></div>
  </section>
</body>
</html>
