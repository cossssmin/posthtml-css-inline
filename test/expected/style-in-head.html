<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px;
    }

    .text-blue {
      color: blue;
    }

    .\@container{
      container-type: inline-size
    }

    div {
      display: flex;
    }

    div {
      font-size: 14px !important;
    }

    @media screen {
      .sm\:w-1\/2 {
        width: 50%;
      }
    }

    @container (min-width: 17.5rem){
      .\@\[17\.5rem\]\:underline{
        text-decoration-line: underline
      }
    }

  </style>
</head>
<body>
  <section class="@container" style="container-type: inline-size">
    <div class="text-blue text-sm @[17.5rem]:underline sm:w-1/2" style="font-size: 14px; color: blue; display: flex"></div>
  </section>
</body>
</html>
